recipes/cgal/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+mkdir -p build && cd build
+
+# needs qt5 for imageio
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DWITH_CGAL_ImageIO=OFF -DWITH_CGAL_Qt5=OFF \
+  ..
+
+make install
+
+
+# language bindings are in a separate repo without releases
+git clone https://github.com/CGAL/cgal-swig-bindings.git
+cd cgal-swig-bindings
+mkdir -p build && cd build
+site_dir=`python -c "from distutils import sysconfig; print(sysconfig.get_python_lib())"`
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DPYTHON_OUTDIR_PREFIX=${site_dir} \
+  -DCOMMON_LIBRARIES_PATH=${PREFIX}/lib \
+  -DBUILD_JAVA=OFF \
+  ..
+make
```

Comment:
An install rule is on the way:
https://github.com/CGAL/cgal-swig-bindings/pull/71
For the tests it's not in good shape either, but I proposed a travis script too.